From d254f7998ef64ac376059eefec6339829b450e71 Mon Sep 17 00:00:00 2001
From: Josh Wu <josh.wu@atmel.com>
Date: Mon, 8 Dec 2014 11:28:54 +0800
Subject: [PATCH] Makefile: remove target 'boot' and fixboot script

To make ROM code load at91bootstrap we need put a the file size in
vector table.
As we use .lds to calcuate the _romsize, and put it in the vector
table of generated binary file.
So we don't need 'boot' target and fixboot script to do this any more.

Signed-off-by: Josh Wu <josh.wu@atmel.com>
Acked-by: Nicolas Ferre <nicolas.ferre@atmel.com>
---
 Makefile           |  8 +-------
 scripts/fixboot.py | 18 ------------------
 2 files changed, 1 insertion(+), 25 deletions(-)
 delete mode 100755 scripts/fixboot.py

diff --git a/Makefile b/Makefile
index 631458b..8e0764d 100644
--- a/Makefile
+++ b/Makefile
@@ -303,13 +303,7 @@ endif
 	@echo "  AS        "$<
 	@$(AS) $(ASFLAGS)  -c -o $@  $<
 
-
-$(AT91BOOTSTRAP).fixboot: $(AT91BOOTSTRAP)
-	./scripts/fixboot.py $(AT91BOOTSTRAP)
-
-boot: $(AT91BOOTSTRAP).fixboot
-
-PHONY+= boot bootstrap
+PHONY+= bootstrap
 
 rebuild: clean all
 
diff --git a/scripts/fixboot.py b/scripts/fixboot.py
deleted file mode 100755
index 3e835a7..0000000
--- a/scripts/fixboot.py
+++ /dev/null
@@ -1,18 +0,0 @@
-#!/usr/bin/env python
-
-import struct, sys
-
-#print sys.argv[1]
-
-fd = open(sys.argv[1], "r")
-
-line = fd.read()
-fd.close()
-
-#print len(line)
-
-vec = struct.pack("<I", len(line))
-
-fd = open(sys.argv[1], "w")
-fd.write(line[0:20]+vec+line[24:])
-fd.close()
-- 
1.9.1

